pkg/k8s/osm_injector_test.go

```diff
@@ -1,102 +0,0 @@
-package k8s
-
-import (
-	"testing"
-
-	tassert "github.com/stretchr/testify/assert"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/kubernetes/fake"
-
-	"github.com/openservicemesh/osm/pkg/constants"
-)
-
-func TestGetOSMInjectorPods(t *testing.T) {
```

Comment:
This test doesn't appear to be replaced

Reply:
The method in the corresponding file was deleted